george/server.js

```diff
@@ -0,0 +1,116 @@
+var express = require('express');
+var app = express();
+var bodyParser = require('body-parser');
+var mongoose = require('mongoose');
+
+mongoose.connect("mongodb://localhost/gorilla_collection");
+var UserSchema = new mongoose.Schema({
+	name: String,
+	age: Number
+});
+mongoose.model('User', UserSchema);
+var User = mongoose.model('User');
+
+app.use(bodyParser.urlencoded({ extended: true }));
+
+var path = require('path');
+
+app.use(express.static(path.join(__dirname, './static')));
+app.set('views', path.join(__dirname, './views'));
+app.set('view engine', 'ejs');
+
+//Get all, root  done!
+app.get('/', function(req, res) {
+	User.find({}, function(err, users) {
+	    if(err) {
+	      console.log('something went wrong');
+	    } else {
+	      console.log(users);
+	      res.render('index', {"users": users});
+	    }
+	  })
+
+    //res.render('index', users);
+})
+
+//show 1 gorilla done!
+app.get('/gorillas/:id',function(req, res){
+	User.find({'_id': req.params.id}, function(err, users){
+		if(err) {
+	      console.log('something went wrong');
+	    } else {
+	      res.render('show.ejs', {"gorilla": users[0]});
```

Comment:
If we use `findOne` instead of `find`, we won't need to deal with the database returning an array!